pwnlib/gdb.py

```diff
@@ -993,10 +993,14 @@ def version(program='gdb'):
 
     Example:
 
-        >>> (7,0) <= gdb.version() <= (8,0)
+        >>> (7,0) <= gdb.version()
         True
     """
-    program = misc.which(program)
```

Comment:
Why are we deleting items out of the environment?  This seems like it will have extremely unintended side-effects.  Neither `PYTHONPATH` nor `PYTHONHOME` have an effect on `misc.which` so I genuinely don't think there's any effect.

Reply:
The side-effects are on `gdb` which uses python3 and breaks if either of these variables are set for python2. This wasn't intended to be upstreamed however.